scripts/simple_sensors_capture.py

```diff
@@ -1,22 +1,28 @@
 import time
+from bson import json_util
```

Comment:
Why not Python's json library?

Reply:
I've been using BSON as it handles some object types better and is "supposed" to be faster while still fully compatible. It's the underlying encoder for mongo. Not sure it's needed here so could change.

Reply:
Specifically, it handles `datetime` encoding without any effort.

Reply:
Can we use bson without installing mongodb?

And does it still produce JSON (text), rather than BSON (binary).

Reply:
The `json_util.dumps` does produce text.

```
{'bar': 42, 'dt': datetime.now(), 'baz': 'bam'}
```
to
```
{"bar": 42, "dt": {"$date": 1516800132526}, "baz": "bam"}
```
Under the hood it is storing the binary representation of `42` as opposed to the string `'42'`, like json. This means the files can actually be larger.